src/equations/compressible_euler_2d.jl

```diff
@@ -865,6 +957,48 @@ end
 end
 
 
+# Compute numerical flux in direction of a normal vector by rotating the velocity, 
+# computing the numerical flux in x-direction, and rotating the calculated flux back.
+@inline function (numflux::FluxPlusDissipation)(u_ll, u_rr, normal_vector::SVector, equations::CompressibleEulerEquations2D)
+  norm_ = norm(normal_vector)
+
+  # cos and sin of the angle between the x-axis and the normalized normal_vector are
+  # the normalized vector's x and y coordinates respectively (see unit circle).
+  c = normal_vector[1] / norm_
+  s = normal_vector[2] / norm_
+
+  # Clockwise rotation by α
+  # Multiply with [ 1     0       0     0;
+  #                 0   cos(α)  sin(α)  0;
+  #                 0  -sin(α)  cos(α)  0;
+  #                 0     0       0     1 ]
+
+  u_ll_rotated = SVector(u_ll[1], 
+                          c * u_ll[2] + s * u_ll[3],
+                         -s * u_ll[2] + c * u_ll[3],
+                         u_ll[4])
+
+  u_rr_rotated = SVector(u_rr[1], 
+                          c * u_rr[2] + s * u_rr[3],
+                         -s * u_rr[2] + c * u_rr[3],
+                         u_rr[4])
+
+  flux_rotated = numflux(u_ll_rotated, u_rr_rotated, 1, equations)
+
+  # Counterclockwise rotation by α
+  # Multiply with [ 1    0        0     0;
+  #                 0  cos(α)  -sin(α)  0;
+  #                 0  sin(α)   cos(α)  0;
+  #                 0    0        0     1 ]
+  flux = SVector(flux_rotated[1], 
+                 c * flux_rotated[2] - s * flux_rotated[3],
+                 s * flux_rotated[2] + c * flux_rotated[3],
+                 flux_rotated[4])
+
+  return flux * norm(normal_vector)
+end
+
+
```

Comment:
Maybe I am missing something, but why didn't you have to implement the `normal_vector::SVector` versions for the flux and the speeds for Euler as you did for LSA?

Reply:
Rotational invariance of Euler allows us to evaluate the numerical flux in x-direction (see above).